h/static/scripts/host-service.coffee

```diff
@@ -0,0 +1,34 @@
+###*
+# @ngdoc service
+# @name host
+#
+# @description
+# The `host` service relay the instructions the sidebar needs to send
+# to the host doument. (As opposed to all guests)
+# It uses the bridge service to talk to the host.
+###
+class HostService
+
+  this.inject = [ '$window', 'bridge' ]
```

Comment:
Can we cut down a bit on whitespace here? `this.inject = ['$window', 'bridge']`.